app/debrief_svc.py

```diff
@@ -57,6 +46,31 @@ async def build_operation_d3(self, operation_ids):
                                                   type='has_agent'))
         return graph_output
 
+    async def build_netpath_d3(self, operation_ids):
+        graph_output = dict(nodes=[], links=[])
+        id_store = dict(c2=0)
+        graph_output['nodes'].append(dict(name="C2 Server", type='c2', label='server', id=0, img='server'))
+        agents = await self.data_svc.locate('agents')
+        self._add_agents_to_d3(agents, id_store, graph_output)
+
+        for op_id in operation_ids:
+            operation = (await self.data_svc.locate('operations', match=dict(id=int(op_id))))[0]
+            for link in operation.chain:
+                if link.ability.tactic == 'lateral-movement' and link.status == 0:
+                    link_graph_id = id_store['link' + link.unique] = max(id_store.values()) + 1
+                    graph_output['nodes'].append(dict(type='latmov_link', name=link.ability.technique_name,
```

Comment:
`latmov_link` might be a confusing name for this node type (since it's a node).

Reply:
Do you think changing it to `type='latmov_step'` might be clearer? Then for operation Link nodes https://github.com/mitre/debrief/blob/07ef293e1bb27877963c93bd5684988fb3508bb5/app/debrief_svc.py#L26 we would want to change those from `type='link'` to `type='step'` as well for consistency?